app/screens/channel/channel.js

```diff
@@ -211,28 +196,24 @@ class Channel extends PureComponent {
                     <OfflineIndicator/>
                     <View style={style.header}>
                         <ChannelDrawerButton/>
-                        <ChannelTitle
-                            onPress={this.goToChannelInfo}
+                        <ChannelTitle onPress={this.goToChannelInfo}/>
+                        <ChannelSearchButton
+                            navigator={navigator}
+                            theme={theme}
                         />
-                        <ChannelSearchButton navigator={navigator}/>
                     </View>
                 </View>
                 <KeyboardLayout
                     behavior='padding'
                     style={style.keyboardLayout}
-                    keyboardVerticalOffset={height}
+                    watchStatusBar={true}
```

Comment:
Looks like this PR shouldn't be merged until the other ones are in

Reply:
yeah I think I've missed that... lets merge the other two and then I can rebase this one

Reply:
Actually, you can remove that line since you removed that prop